app/konnect/private-connections/aws-privatelink.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Create a private connection with AWS PrivateLink
-subtitle: Connect your data plane to your Konnect control plane with a private connection to stay compliant and save data transfer costs
+subtitle: Connect your data plane to your Konnect control plane or make Konnect Admin API calls with a private connection to stay compliant and save data transfer costs
```

Comment:
```suggestion
subtitle: Connect your data plane to your Konnect control plane or make Konnect API calls (including Admin API calls for control plane configuration) with a private connection to stay compliant and save data transfer costs
```

Reply:
Adding a differentiator for Konnect API calls and Admin API calls as the term Admin API is used very closely with CP configuration only.